lms/djangoapps/mobile_api/utils.py

```diff
@@ -5,6 +5,7 @@
 API_V05 = 'v0.5'
 API_V1 = 'v1'
 API_V2 = 'v2'
+API_V3 = 'v3'
```

Comment:
This isn't necessarily your problem but do we know what is still using the older versions of this API.  I don't think it should block your work, but it would be useful to drop support for the older versions and I'm trying to understand if there is a way to know what's using the older APIs.

Reply:
As per my understanding, there are still plenty of Open edX operators who are using custom developed mobile apps based on the older versions of Open edX mobile applications with newer versions of the edx-platform.
Several of our clients benefited from this during the version upgrade, because it wasn't required to update mobile apps.
Not sure if v0.5 or v1 are still used in the latest versions of mobile apps.